### PR TITLE
Add the largest heading to the title

### PR DESCRIPTION
**Pull request by tikipatel**

### CODE REVIEW

**Comment by gertschmeghan** [comment]
This is not how you make the largest heading. The largest heading is ```<h1>My Web Page</h1>```

File: index.html [hunk 1]
```diff
@@ -1 +1 @@
-My Web Page
+<h6>My Web Page</h6>
```
Comment (shandysulen) [inline index.html]: Hey this is how you make the smallest heading! You can create the largest heading instead with:
```
<h1>My Web Page</h1>
```

Comment (clairegraybeal) [inline index.html]: The way to make the largest headline would actually be:

```<h1>My Web Page</h1>```

Try that!

Comment (Uniquebydesign) [inline index.html]: <h1>My Web Page</h1>

Comment (Uniquebydesign) [reply index.html]: `<h1>My Web Page</h1>`

Comment (Uniquebydesign) [reply index.html]:
```
The correct way to make largest heading
<h1>My Web Page</h1>
```

**Comment by tikipatel** [comment]
Heard you all! Changing now...

**Pull request merged**